src/plugins/tabs/components/tabs.vue

```diff
@@ -156,6 +173,12 @@ export default {
         },
         isCurrent(tab) {
             return _.isEqual(this.currentTab, tab)
```

Comment:
This might be a good time to remove this usage of lodash

Reply:
good catch, removed the dependency and used a more performant check